build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
     api "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}"
     implementation "com.android.support:support-annotations:${versions.supportLibrary}"
     implementation "com.android.support:support-compat:${versions.supportLibrary}"
+    implementation "com.android.support:design:${versions.supportLibrary}"
```

Comment:
This library isn't taking on additional dependencies. It focuses on platform APIs only.

Reply:
Ok, thanks